cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/sqlite/SQLDatabaseQueue.java

```diff
@@ -189,4 +195,30 @@ public boolean isShutdown() {
             throw new RejectedExecutionException("Database is closed");
         }
     }
+
+    /**
+     * Returns the SQLite Version.
+     * @return The SQLite version or "Unknown" if the version could not be determined.
+     */
+    public String getSQLiteVersion() {
```

Comment:
I'd prefer to see this run once when opening the database (avoiding the need to queue it) and then storing the version as a constant for any use later. We could include it in the thread name for instance.